src/python/py/models/quantized_model.py

```diff
@@ -334,23 +371,37 @@ def __init__(self, quant_type, input_path, bits, group_size, q_size, kv_size, in
                             # model.layers.layer_id.mlp.dense_h_to_4h.bias
                             module.mlp.gate_proj.bias = tensor[: intermediate_size]
                             module.mlp.down_proj.bias = tensor[intermediate_size: ]
+                        elif bool(re.match(r"^model.layers\.\d+\.mlp.down_proj.lora_A\.weight$", name)):
+                            module.mlp.down_proj.lora_A.weight = tensor
+                        elif bool(re.match(r"^model.layers\.\d+\.mlp.down_proj.lora_B\.weight$", name)):
+                            module.mlp.down_proj.lora_B.weight = tensor
+                        elif bool(re.match(r"^model.layers\.\d+\.mlp.up_proj.lora_A\.weight$", name)):
+                            module.mlp.up_proj.lora_A.weight = tensor
+                        elif bool(re.match(r"^model.layers\.\d+\.mlp.up_proj.lora_B\.weight$", name)):
+                            module.mlp.up_proj.lora_B.weight = tensor
+                        elif bool(re.match(r"^model.layers\.\d+\.mlp.gate_proj.lora_A\.weight$", name)):
+                            module.mlp.gate_proj.lora_A.weight = tensor
+                        elif bool(re.match(r"^model.layers\.\d+\.mlp.gate_proj.lora_B\.weight$", name)):
+                            module.mlp.gate_proj.lora_B.weight = tensor
+                        elif bool(re.match(r"^model.layers\.\d+\.self_attn.k_proj.lora_A\.weight$", name)):
+                            module.self_attn.k_proj.lora_A.weight = tensor
+                        elif bool(re.match(r"^model.layers\.\d+\.self_attn.k_proj.lora_B\.weight$", name)):
```

Comment:
Can we re-order the cases to the following based on appearance in the ONNX model?

1. `q_proj`
2. `k_proj`
3. `v_proj`
4. `o_proj`
5. `gate_proj`
6. `up_proj`
7. `down_proj`

Can we also add a comment in each case that specifies what the pattern matching is attempting to find (for example, `model.layers.layer_id.mlp.down_proj.lora_A.weight` for the first one)?